src/components/List/RowLabel.js

```diff
@@ -78,9 +78,16 @@ const DesktopWrapper = styled.div`
 const LabelText = styled(Text)`
   font-size: ${typography.size.uno};
   font-weight: ${typography.weight.semiBold};
-  ${({ sentiment }) =>
-    sentiment === "positive" &&
-    `color: ${themes.global.special.base};`} text-transform: uppercase;
+  ${({ sentiment }) => {
```

Comment:
Same comment regarding using `colorThemes` instead of `colors` applies here and inside `renderLabel`.  You might want to consider extracting this into a `css` mixin.